src/WebsiteProvider/Api/MappingHandlers.cs

```
@@ -25,6 +25,11 @@ public void Register()
 
         public void MapPinningRule(WebMap webMap, string registeredSectionUri = null)
         {
+            if (webMap.Section == null)
```

I think this case should be unacceptable and the method should throw an exception when this condition returns true. User should not be able to save such pinning rules. WDYT?